src/Service.GraphQLBuilder/GraphQLTypes/SupportedTypes.cs

```diff
@@ -8,6 +8,7 @@ namespace Azure.DataApiBuilder.Service.GraphQLBuilder.GraphQLTypes
     /// </summary>
     public static class SupportedTypes
     {
+        public const string UUID_TYPE = "UUID";
```

Comment:
Do we need to remove GUID_TYPE from this list? i'm a little confused on the mixed usage of `UUID_TYPE` and `GUID_TYPE`

Reply:
Removed. GUID_TYPE was only being used to identify the column name "guid_types" in the test type_table. Renamed the column to uuid_types to avoid confusion. But essentially, GUID_TYPE is equivalent to UUID_TYPE